### ISSUE TITLE
Error handle in `smart_round()`

### ISSUE THREAD

**Issue by bstaton1** [bstaton1/KuskoHarvUtils#5]
This frequently comes up:

```
Error in checkHT(n, dx <- dim(x)) :
invalid 'n' - must contain at least one non-missing element, got none.
```

And can be reproduced with 

```R
KuskoHarvUtils::smart_round(c(NA, NA))
```

It would be really useful for users if something like this was returned instead:

```
Error in smart_round(): no non-missing elements in x
```

**Comment by bstaton1** [comment]
Actually, upon second thought, it makes more sense to just return all `NA` elements as provided to the function, and only evaluate the smart rounding if possible.

**Comment by bstaton1** [comment]
Closed by #6
